proposal.rst

```diff
@@ -184,14 +206,18 @@ Motivations
 
 **Why have you chosen this particular project?**
 
-FILLME
+The project involves coding in C++ which I use the most while programming and
+would help me increase my knowledge about Linux.
+
 
 **Who will benefit from your project and in what ways?**
 
 .. For example, think about the likely user-base, what they currently have to
 .. do and how your project will improve things for them.
 
-FILLME
+This project removes dependencies on external software, adds support where it
```

Comment:
You don't mention in your proposal where you'd *remove* dependencies on external software, and the project outline we provide doesn't mention it either. You might change some dependencies for others, is that what you mean? It's worth taking the time to be as clear as possible in your proposal.